styles/books/chemistry/book.scss

```
@@ -129,8 +147,8 @@ $bandBorder: #{$bandSize + $fullBleedWidth} solid $bandColor;
   margin-#{$side}: -1 * $fullBleedWidth;
 }
 
-@mixin setEocPageBorder($type, $side) {
-  @page eoc:#{$type} {
+@mixin setPageBorder($paget-type, $type, $side) {
```

```suggestion
@mixin setPageBorder($page-type, $type, $side) {
```

Was this working? When I switch branches, I get a compilation error when I try to build the styles myself because of this typo.

---

I fixed the typo.